lib/tasks/foreman_remote_execution_tasks.rake

```diff
@@ -38,8 +38,3 @@ namespace :foreman_remote_execution do
 end
 
 Rake::Task[:test].enhance ['test:foreman_remote_execution']
-
-load 'tasks/jenkins.rake'
-if Rake::Task.task_defined?(:'jenkins:unit')
-  Rake::Task['jenkins:unit'].enhance ['test:foreman_remote_execution', 'foreman_remote_execution:rubocop']
```

Comment:
My understanding of the linked issue was that only rubocop is problematic, not tests. Why remove both?